x-pack/platform/plugins/shared/task_manager/server/integration_tests/managed_configuration.test.ts

```diff
@@ -130,8 +136,10 @@ describe('managed configuration', () => {
       taskManagerStart = await taskManager.start(coreStart, {});
 
       // force rxjs timers to fire when they are scheduled for setTimeout(0) as the
-      // sinon fake timers cause them to stall
-      clock.tick(0);
+      // sinon fake timers cause them to stall. We need to do this a few times for the
+      // startup code to start monitoring the poll configuration properly.
+      await runSetTimeout0();
+      await runSetTimeout0();
```

Comment:
We now need two event loop cycles: first to resolve the promise for getting the esClient, second to resolve the promise for the health API call to resolve.